lib/perl/Genome/Model/Tools/EpitopePrediction/NewPipeline.pm

```diff
@@ -0,0 +1,192 @@
+package Genome::Model::Tools::EpitopePrediction::NewPipeline;
```

Comment:
The new system doesn't support bed files which is currently still a feature used by some analysts. Ultimately we want to move to this piepleine exclusively but it might require adding on some pre-processing of bed to files to create vcf files that can be used in the variant reporting system. Once that is hall hashed out the old pipeline should go away and this can be renamed.